src/Resources/config/services/events/workflow.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <services>
+        <defaults autoconfigure="true" autowire="true" public="true" />
```

Comment:
```suggestion
        <defaults autoconfigure="true" autowire="true" public="true" />
```

It shouldn't be here. `autoconfigure` and `autowire` is not recommended for bundles, `public` is a legacy thing, that shouldn't be present in 99% cases.